lib/components/overlays/quick_settings/quick_settings_overlay.dart

```diff
@@ -117,6 +125,69 @@ class _QuickSettingsOverlayState extends State<QuickSettingsOverlay>
   }
 }
 
+PageRouteBuilder customRouteTransition(Widget screen) {
+  return PageRouteBuilder(
+    transitionDuration: const Duration(milliseconds: 150),
+    reverseTransitionDuration: const Duration(milliseconds: 150),
+    pageBuilder: (context, animation, secondaryAnimation) => screen,
+    transitionsBuilder: (context, animation, secondaryAnimation, child) {
+      final fadeAnimationIn = Tween(
+        begin: 0.0,
+        end: 1.0,
+      ).animate(
+        CurvedAnimation(
+          parent: animation,
+          curve: Curves.linear,
```

Comment:
If curve is linear you can just pass the actual animation controller, nothing will change, this applies to every instance in this block